src/Fixer/Import/GlobalNamespaceImportFixer.php

```diff
@@ -119,27 +129,35 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
         $newImports = [];
 
         if (true === $this->configuration['import_constants']) {
-            $newImports['const'] = $this->importConstants($tokens, $useDeclarations);
+            $newImports['const'] = array_values($this->importConstants($tokens, $useDeclarations));
```

Comment:
Can we instead of this and 2 below add `array_values` once in `prepareImports` method?

Reply:
I can't add it to `prepareImports()` because the result of that method is merged with other array in `importClasses()` and at that point keys are important. But I think I can remove `array_values()` from here, because string keys are  allowed in `ImportProcessor::insertImports()` (at some point I had only `list<class-string>` there, so probably it's a leftover from refactoring.